zerver/openapi/zulip.yaml

```diff
@@ -20733,6 +20741,42 @@ paths:
             schema:
               type: object
               properties:
+                target_users:
+                  description: |
+                    An object specifying the users and user groups whose settings an
+                    administrator intends to modify. It contains two optional fields
+                    `user_ids` and `group_ids`. This parameter is only applicable when
+                    the requesting user has realm owner privileges, enabling updates
+                    across multiple users or group members.
```

Comment:
"This parameter is only applicable" is weird/wrong here -- the point is that this is how you **request** doing that.